test/data/mock-send-state.json

```diff
@@ -24,6 +25,9 @@
         "name": null
       }
     },
+    "showIpfsModalOpen": false,
+    "showKeyringRemovalSnapModal": false,
+    "showWhatsNewPopup": false,
```

Comment:
just curious, seems like a there are few things added here that are unrelated to the gasFee polling, is there a reason they were needed as part of this PR?

Reply:
wrapping renders in `act` catches previous uncaught state selection failures